### PR TITLE
[SC] Upgrade Stratis.SmartContracts package to 1.2.1.

### PR DESCRIPTION
**Pull request by codingupastorm**

### CODE REVIEW

File: src/Stratis.SmartContracts.CLR.Tests/AuctionTests.cs [hunk 1]
```diff
@@ -2,6 +2,7 @@
 using System.Collections.Generic;
 using Stratis.SmartContracts.CLR.Serialization;
 using Stratis.SmartContracts.Networks;
+using Stratis.SmartContracts.RuntimeObserver;
```
Comment (fassadlr) [inline src/Stratis.SmartContracts.CLR.Tests/AuctionTests.cs]: How come this was added without an apparent use for it in the file? Was the project not building or something?

File: src/Stratis.SmartContracts.IntegrationTests/ContractExecutionFailureTests.cs [hunk 2]
```diff
@@ -13,6 +13,7 @@
 using Stratis.SmartContracts.CLR.Serialization;
 using Stratis.SmartContracts.Core;
 using Stratis.SmartContracts.Core.Util;
+using Stratis.SmartContracts.RuntimeObserver;
```
Comment (fassadlr) [inline src/Stratis.SmartContracts.IntegrationTests/ContractExecutionFailureTests.cs]: Same here...

File: src/Stratis.SmartContracts.CLR.Tests/AuctionTests.cs (hunk 1 above)
Comment (codingupastorm) [reply src/Stratis.SmartContracts.CLR.Tests/AuctionTests.cs]: `Stratis.SmartContracts.RuntimeObserver.IGasMeter` and `Gas` replaced `Stratis.SmartContracts.Gas`

Comment (codingupastorm) [reply src/Stratis.SmartContracts.CLR.Tests/AuctionTests.cs]: Identical code, just got moved

**Pull request merged**